client/logclient_test.go

```diff
@@ -712,3 +721,69 @@ func TestGetAcceptedRootsErrors(t *testing.T) {
 		}
 	}
 }
+
+func TestGetEntryAndProof(t *testing.T) {
+	hs := serveRspAt(t, "/ct/v1/get-entry-and-proof", GetEntryAndProofResp)
+	defer hs.Close()
+	lc, err := client.New(hs.URL, &http.Client{}, jsonclient.Options{})
+	if err != nil {
+		t.Fatalf("Failed to create client: %v", err)
+	}
+
+	tests := []struct {
+		index    uint64
+		treesize uint64
+	}{
+		{1000, 2000},
+	}
+
+	for _, test := range tests {
+		resp, err := lc.GetEntryAndProof(context.Background(), test.index, test.treesize)
+		if err != nil {
+			t.Errorf("GetEntryAndProof(%v, %v)=nil,%v; want proof,nil", test.index, test.treesize, err)
+		} else if got := len(resp.AuditPath); got < 1 {
+			t.Errorf("len(GetEntryAndProof(%v, %v)): %v; want > 1", test.index, test.treesize, got)
+		}
+	}
```

Comment:
Do you need to check validity of the entry and proof?

Reply:
Dunno. I just copied the other tests in here.